pkg/minikube/config/config.go

```diff
@@ -88,11 +88,7 @@ func WriteConfig(configFile string, m MinikubeConfig) error {
 }
 
 // ReadConfig reads in the JSON minikube config
-func ReadConfig() (MinikubeConfig, error) {
-	return readConfig(constants.ConfigFile)
-}
-
-func readConfig(configFile string) (MinikubeConfig, error) {
+func ReadConfig(configFile string) (MinikubeConfig, error) {
```

Comment:
I wonder, could this name be changed to something more obvious that this is not a profile config but global minikube config ?